csharp/ql/lib/semmle/code/csharp/Type.qll

```diff
@@ -765,6 +789,18 @@ class Class extends RefType, @class_type {
   override string getAPrimaryQlClass() { result = "Class" }
 }
 
+/**
+ * DEPRECATED: Use `RecordClass` instead.
+ */
+deprecated class Record extends Class {
+  Record() { this.isRecord() }
+
+  /** Gets the clone method of this record. */
+  RecordCloneMethod getCloneMethod() { result = this.getAMember() }
+
+  override string getAPrimaryQlClass() { result = "Record" }
```

Comment:
Remove; we do don't want to encourage the use of deprecated classes.

Reply:
Alright - I will kill the override.